core/src/main/scala/org/apache/spark/sql/delta/hooks/DoAutoCompaction.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
```

Comment:
2022 at the very least IMHO

Reply:
Let me keep that now for scalastyle check